pkg/collector/python/init.go

```diff
@@ -164,6 +165,8 @@ void initkubeutilModule(rtloader_t *rtloader) {
 */
 import "C"
 
+const PythonWinExeBasename = "python.exe"
```

Comment:
I agree. I think this module would ideally do good with a split-out of some things into the platform-specific modules like you say but at current time it would have been a much bigger change than desired, especially since we play with module-level vars in the affected methods. I can keep that in mind for future work on this. Thoughts?

Reply:
Yeah, no worries, I don't think you need to do this here. We should maybe create a card to try and remember to do that. It may take some time as it'd just be a refactor, but probably still worth it.